allspice/utils/netlist_generation.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+import time
+from typing import Union
+
+from ..allspice import AllSpice
+from ..apiobject import Content, Ref, Repository
+from ..exceptions import NotYetGeneratedException
+
+
+@dataclass
+class PcbComponent:
+    designator: str
+    pins: list[ComponentPin]
+
+
+@dataclass
+class ComponentPin:
+    designator: str
+    net: str
+
+
+@dataclass
+class NetlistEntry:
+    net: str
+    pins: list[str]
+
+
+Netlist = list[str]
+
+
+def generate_netlist_for_altium(
```

Comment:
Is there a reason this wouldn't be the same for non altium pcbs?

Reply:
Nope. Moving to a common module. ✔️